src/test/regress/expected/multi_name_lengths.out

```diff
@@ -209,18 +266,13 @@ SELECT master_create_worker_shards('sneaky_name_lengths', '2', '2');
 
 \c - - :public_worker_1_host :worker_1_port
 \di public.sneaky*225006
-                                                    List of relations
- Schema |                              Name                               | Type  |  Owner   |           Table
+          List of relations
```

Comment:
Hmm, why these tests changed? I couldn't follow

Reply:
Now that I added more queries in this sql file, it appears that the shardId values are now different.
I should update many shardId occurences to keep the outputs the same as before.